homeassistant/components/imap/coordinator.py

```diff
@@ -94,6 +94,23 @@ async def connect_to_server(data: Mapping[str, Any]) -> IMAP4_SSL:
     return client
 
 
+def _decode_payload(part: Message) -> str:
```

Comment:
Alternatively this could be a static method on the ImapMessage class.

Reply:
done